htmd/builder/builder.py

```diff
@@ -155,6 +157,15 @@ def embed(mol1, mol2, gap=1.3):
     return mol2
 
 
+def convertDisulfide(mol, disu):
```

Comment:
Following my remark above, maybe we want to extend this function to a `validateDisulfide()` function, where we both convert and validate if the passed selections are valid for disulfide bonds? Maybe not erroring (the user may want to do disulfide bonds between a cysteine-like residue which has a weird name), but warn the user is the residue does not have a sulphur and resname does not start with CY*?

Reply:
Can we do this in a FF generic way? I was planning to do it on the individual builder layer where I know the exact name of the sulfur in a cysteine but if we can assume that you need a single `S*` atom in a `CY*` residue it would also work. But it would not be as exact as you could have some stupid stuff like `SC` instead of `SG` which would crash `tleap`. Maybe better do the check on the individual builders?

Reply:
well, we have the element guessing for sulphur (right?) and I think CY* should work on both FFs (except of the case I already mentioned of cysteine'like residues with weird naming)

Reply:
The convert is only there until we deprecate the old way of using the disulfides. Also as I said I'd prefer not to check for the existence of atoms.

Reply:
I disagree. You still need to convert the user-passed lists to `UniqueResidueID`. Anyway, some refactoring that can be left for later.